### ISSUE TITLE
После логаута я не могу снова залогиниться, кнопка неактивна

### ISSUE THREAD

**Issue by andrewQwer** [Joiler/SimpleChat#3]
![image](https://cloud.githubusercontent.com/assets/710513/7387708/fb46a964-ee67-11e4-8001-0f8ca76d52bc.png)

**Comment by andrewQwer** [comment]
разобрался, присутствует минимально необходимое количество символов для ввода. Было бы неплохо уведомить пользователя, что необходимо ввести не меньше n символов.

**Comment by andrewQwer** [comment]
экспериментальным путем установлено, что это действует только на пароль, логин может быть и 1 символ.

**Comment by Joiler** [comment]
Да, не спорю, незнающему пользователю может показаться странно. Добавил подстветку input-ов при валидации.

**Comment by andrewQwer** [comment]
стало чуточку понятнее :)
